docs/MaintainerVersioning.md

```diff
@@ -242,7 +138,7 @@ BreakingChangeUpdate.x OR BreakingChangeUpdate.Enhancement.x
 2.x OR 2.2.x
 ```
 
-At the time that this document was published, Zowe CLI was at version `1.0.7` and Imperative was at `1.0.8`. Both packages' peerDependencies versions in a supported plug-in can be specified as `1.x`. This indicates that we anticipate that the plug-in will work with all upcoming bug fixes and with all new features that do not break backward-compatibility, such as `1.1.0`, or `1.4.6`, but not with `2.0.0` or greater). By specifying a version of `1.x`, the plug-in does not have to update its version string and republish frequently just to avoid compatibility warnings when patches and backward-compatible enhancements of Zowe CLI or Imperative are published. For a supported plug-in, this ties the plug-in to a reasonable range of non-breaking changes in Zowe CLI and Imperative. By the time that the next feature release of the plug-in be published, either Zowe CLI or Imperative might be specified as `2.x` and you would lock in that level of releases for the life of that supported version of the plug-in.
+At the time that this document was published, Imperative was at version `1.0.8`. Imperative's peerDependencies version in a supported plug-in can be specified as `1.x`. This indicates that we anticipate that the plug-in will work with all upcoming bug fixes and with all new features that do not break backward-compatibility, such as `1.1.0`, or `1.4.6`, but not with `2.0.0` or greater). By specifying a version of `1.x`, the plug-in does not have to update its version string and republish frequently just to avoid compatibility warnings when patches and backward-compatible enhancements of Imperative are published. For a supported plug-in, this ties the plug-in to a reasonable range of non-breaking changes in Zowe CLI and Imperative. By the time that the next feature release of the plug-in be published, Imperative might be specified as `2.x` and you would lock in that level of releases for the life of that supported version of the plug-in.
```

Comment:
- Please use backticks for peerDependencies
- Either remove the single close parenthesis, or add an open parenthesis
- I think there's a word missing here: "By the time that the next feature release of the plug-in be published ... "